src/main/java/models/SpanishGame.java

```diff
@@ -0,0 +1,106 @@
+package models;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Random;
+
+
+public class SpanishGame extends Game {
+
+	public SpanishGame(){
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+        cols.add(new ArrayList<Card>());
+	}
+
+
+    public void buildDeck() {
+        for(int i = 1; i < 11; i++){
+            deck.add(new Card(i, Suit.clubs));
+            deck.add(new Card(i, Suit.hearts));
+            deck.add(new Card(i, Suit.diamonds));
+            deck.add(new Card(i, Suit.spades));
+        }
+    }
+
+
+    public void shuffle() {
```

Comment:
Do we need shuffle to be defined in each of the classes? It looks like its the same for both American and Spanish.